ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/configs/DynamoDBConfigs.java

```diff
@@ -37,24 +39,44 @@
 	 * a fundamental data element, something that does not need to be broken down
 	 * any further.
 	 */
-	@DefaultValue("name")
+	@PropertyName(name = "attributename")
+	@DefaultValue("id")
 	String getAttributeName();
 
 	/*
 	 * Used for provisioned throughput
 	 */
-	@DefaultValue("1")
+	@PropertyName(name = "rcu")
+	@DefaultValue("2")
 	String getReadCapacityUnits();
 
-	@DefaultValue("1")
+	@PropertyName(name = "wcu")
+	@DefaultValue("2")
 	String getWriteCapacityUnits();
 
 	/*
 	 * Consistency: When you request a strongly consistent read, DynamoDB returns a
 	 * response with the most up-to-date data, reflecting the updates from all prior
 	 * write operations that were successful.
 	 */
+	@PropertyName(name = "consistentread")
 	@DefaultValue("false")
 	Boolean consistentRead();
 	
+	/*
+	 * This configuration allows to create a table programmatically (through NDBench).
+	 * In the init phase, we create a table. 
+	 * In the shutdown phase, we delete the table.
+	 * 
+	 * In a single node case it works fine. In a multi-node deployment, there 
+	 * are a number of race conditions.
+	 * * The first instance will create the table. It does not matter too much which does it
+	 * because we create the table only if does not exist.
+	 * * The first instance will delete the table. It does not matter too much which does it
+	 * because the table will be eventually deleted. All others will show exceptions.
+	 * 
+	 */
+	@DefaultValue("false")
+	Boolean programTables();
```

Comment:
`programTables` is used both at the table creation and deletion. My thought process is programmatically updating tables instead of through the AWS console, therefore `programTables`.

Reply:
programTables is definitely less obvious?
Should deletetables be a seperate option? One might want to run tests on large data and you do not have to delete and re-run the tests entirely with empty table everytime.

Reply:
There is an action to delete a table when you have set this property in the shutdown. In regards to the second part of your comment, yes this is true that is why if you want to test with a large amount of data you are not going to use this option.